### PR TITLE
Create standalone send page - Closes #1495 

### LINKED ISSUE #1495: Create standalone Send page
Subtask of #1466

The goal of this ticket is to create standalone Send page that can be accessed by path `/wallet/send`, is accessible only after login and in another subtask of #1466 will be linked from Wallet page

### PR DESCRIPTION
**Pull request by osvaldovega**
### What issue have I solved?

-- #1495

### How have I implemented/fixed it?

Working on it.
Standalone page created, right now navigation is possible and route file updated to properly use /send in browser.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

**Comment by osvaldovega** [comment]
requested changes are fixed, branch updated

File: src/components/multiStep/index.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ class MultiStep extends React.Component {
       prevStep: step.prevStep,
       reset: this.reset.bind(this),
       ...step.data[step.current],
+      finalCallback,
```
Comment (slaweet) [inline src/components/multiStep/index.js]: If you add `finalCallback` here, the condition `if (step.current === (children.length - 1)) {` below is not needed

File: src/components/resultBox/resultBox.js [hunk 2]
```diff
@@ -45,29 +45,17 @@ class ResultBox extends React.Component {
...
-              this.props.history.replace(this.props.history.location.pathname);
-            }}>
-            {this.props.t('Okay')}
-          </Button>
           {this.props.success &&
-            this.props.reciepientId && this.isNotYetFollowed(this.props.reciepientId) ?
+            this.props.recipientId && this.isNotYetFollowed(this.props.recipientId) ?
             <Button className={`add-follwed-account-button ${styles.addFollowedAccountButton}`}
               onClick={() => {
                 this.props.nextStep({ address: this.props.reciepientId });
```
Comment (slaweet) [inline src/components/resultBox/resultBox.js]: The type needs to be fixed on this line too. Currently, the followed account is saved without the address.

```suggestion
                 this.props.nextStep({ address: this.props.recipientId });
```

File: src/components/multiStep/index.js (hunk 1 above)
Comment (osvaldovega) [reply src/components/multiStep/index.js]: you are right it is fixed now

File: src/components/resultBox/resultBox.js (hunk 2 above)
Comment (osvaldovega) [reply src/components/resultBox/resultBox.js]: fixed

**Pull request merged**